operations/accountnotes.md

```diff
@@ -0,0 +1,345 @@
+                }
+            },
+            "UpdaterProfile": {
+                "Discriminator": "Integration",
+                "EnterpriseProfile": null
+            }
+        },
+        {
+            "Id": "da34b396-41f7-47f6-8847-aecf00a3f19e",
+            "AccountId": "5fcd1933-22f2-40b9-84da-7db04cbecec2",
+            "Content": "Lactose intolerant",
+            "AccountType": "Customer",
+            "Classifications": ["FoodAndBeverage"],
+            "IsActive": true,
+            "CreatorProfile": {
+                "Discriminator": "Enterprise",
+                "EnterpriseProfile": {
+                    "ProfileId": "52d19c34-b0aa-4635-905d-1326fa8b8e13"
+                }
+            },
+            "UpdaterProfile": {
+                "Discriminator": "Integration",
+                "EnterpriseProfile": null
+            }
+        }
+    ],
+    "Cursor": "da34b396-41f7-47f6-8847-aecf00a3f19e"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AccountNotes` | array of [Account note](#account-note) | required | The account note of the account within the chain. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### Account note
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the account note. |
+| `AccountId` | string | required | Unique identifier of the account. |
+| `Content` | required | required | The content of the account note. |
+| `AccountType` | string [Account type](#account-type)| required | Specifying types of accounts provided ([Customer](../operations/customers.md#customer), [Company](../operations/companies.md#company)). |
+| `Classifications` | array of string | required | Specifying the classifactions of the note based on account type ([Customer note classification](#customer-note-classifications), [Company note classifications](#company-note-classifications)). |
+| `IsActive` | boolean | required | Whether the account note is till active. |
+| `CreatorProfile` | [Profile data](#profile-data) | required | The profile data of the user who created the account note. |
+| `UpdaterProfile` | [Profile data](#profile-data) | required | The profile data of the user who updated the account note. |
+
+#### Customer note classifications
+
+* `General`
+* `FoodAndBeverage`
+* `FrontOffice`
+* `Reservations`
+* `Housekeeping`
+* `Maintenance`
+* `PreviousStay`
+* `FamilyRelations`
+* `Gifts`
+* `Accounting`
+* `Complaints`
+* `Other`
+
+#### Company note classifications
+
+* `General`
+
+## Add account notes
+
+> This feature is being actively developed, features and behavior of this operation may change at short notice. Currently we support only one classification per account note.
```

Comment:
```suggestion
> ### Restricted!
> This feature is being actively developed, features and behavior of this operation may change at short notice. Currently we support only one classification per account note.
```

Reply:
About this, I think we no longer need to add the message about "being actively developed" 🤔 
As far as I know, we don't have any upcoming change. I think we can get rid of that part completely. @jonas-paul that works for you too?